units/crons.go

```diff
@@ -329,9 +329,8 @@ func PopulateIdleHostJobs(env evergreen.Environment) amboy.QueueOperation {
 		}
 
 		for _, info := range distroHosts {
-			distroID := info.DistroID
 			totalRunningHosts := info.RunningHostsCount
-			minimumHosts := distrosMap[distroID].PlannerSettings.MinimumHosts
+			minimumHosts := distrosMap[info.DistroID].PlannerSettings.MinimumHosts
```

Comment:
Is it possible to add a test that covers this bug?

Reply:
Working on it

Reply:
Added a test for initial review.